src/ocm/components/clusterConfiguration/staticIp/components/FormViewNetworkWide/FormViewNetworkWideFields.tsx

```diff
@@ -207,7 +207,6 @@ export const FormViewNetworkWideFields: React.FC<{ hosts: FormViewHost[] }> = ({
             name="vlanId"
             isRequired
             data-testid="vlan-id"
-            type={TextInputTypes.number}
```

Comment:
If you are removing the input type, you probably need to put a regex in the validation schema or something. Yup takes `12e2` as a valid number but doesn't convert it to `12e2 = 12*10^2 = 1200` like the input field does.

![image](https://user-images.githubusercontent.com/87187179/190353473-83449898-f610-4657-8278-d6e7b1697280.png)

Reply:
Ok, I'll review it

Reply:
I add a new validation to avoid exponential numbers.
1e2 = 100, for example
Now, in this field we have two validations: one for detect if all numbers are digits and other to detect if a number is exponential

Reply:
Something is not right here... @ammont82 

![image](https://user-images.githubusercontent.com/87187179/190418856-3661fac8-893d-41e9-84ea-56cffe694a2f.png)

```
#generated by ui form view
#ipv4MachineNetwork 192.0.0.0/23
interfaces:
  - name: DUMMY4
    type: ethernet
    state: up
  - name: DUMMY4.100
    type: vlan
    state: up
    vlan:
      base-iface: DUMMY4
      id: '100'
    ipv4:
      address:
        - ip: 0.0.0.0
          prefix-length: 24
      enabled: true
      dhcp: false
dns-resolver:
  config:
    server:
      - 192.0.0.1
routes:
  config:
    - destination: 0.0.0.0/0
      next-hop-address: 192.0.0.2
      next-hop-interface: DUMMY4.100
      table-id: 254
```

Reply:
I've changed it and resolved the problem